config/routes.rb

```diff
@@ -156,6 +157,7 @@
       resources :recruitment_cycles,
                 only: %i[index show],
                 param: :year do
+        resources :organisations, only: :index
```

Comment:
Do we actually need / want this endpoint?

Reply:
I wasn't 100% sure since orgs aren't cycle aware. Makes it simpler to grab params[:recruitment_cycle] if needed if it does though.

Reply:
have removed it 👍